app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageTypes.kt

```diff
@@ -80,29 +87,90 @@ internal fun MessageBody(messageBody: MessageBody, onLongClick: (() -> Unit)? =
 fun MessageImage(
     rawImgData: ByteArray?,
     imgParams: ImageMessageParams,
+    assetUploadStatus: Message.UploadStatus,
     onImageClick: Clickable,
 ) {
+    val imageData: Bitmap? = if (rawImgData != null && rawImgData.size < MessageComposerViewModel.IMAGE_SIZE_LIMIT_BYTES)
```

Comment:
We're not supposed to show larger images?

Reply:
Not really. It was like a non written convention/internal agreement that handling & displaying images larger than 15MB was not supported. Even on the electron client is implemented like that.

<img width="709" alt="Captura de pantalla 2022-09-16 a las 16 12 01" src="https://user-images.githubusercontent.com/2468164/191228106-213ddb6f-72da-454a-a510-b100f78ad04d.png">

In any case, this would not be a possible scenario as we are also showing an error message on the `MessageComposerViewModel` when the user selects an image larger than 15MB.